Code/utility_functions.py

```diff
@@ -210,3 +218,20 @@ def last_minima(self, all_data):
     min_times = time[angle_max_index]
 
     return min_times
+
+def total_angle(be, se0, se1):
+    """
+    Takes encoder values at a single time and converts to a total angle.
+    Used for triple pendulum examples.
+    """
+    x, y = position_seat_cartesian(be, se0, se1)
+    total_angle = np.arctan(x/y)
```

Comment:
do we want it returned in radians or degrees? I think degrees as this should be similar to the big encoder values.

Reply:
all right big man I'll fix these later